packages/nextjs/app/builders/0x04cdfb463600a29e8e93053b88daa11f54379907/page.tsx

```diff
@@ -0,0 +1,160 @@
+        <div className="absolute top-0 right-0 w-96 h-96 bg-blue-500 rounded-full mix-blend-multiply filter blur-3xl opacity-20 animate-blob"></div>
+        <div className="absolute -bottom-8 left-20 w-96 h-96 bg-purple-500 rounded-full mix-blend-multiply filter blur-3xl opacity-20 animate-blob animation-delay-2000"></div>
+      </div>
+
+      {/* Content */}
+      <div className="relative z-10 flex items-center justify-center min-h-screen px-4 sm:px-6 lg:px-8">
+        <div className="w-full max-w-2xl">
+          <div className="bg-slate-800/50 backdrop-blur-xl border border-slate-700/50 rounded-2xl p-8 sm:p-12 shadow-2xl">
+            {/* Header */}
+            <div className="text-center mb-8">
+              <div className="inline-flex items-center justify-center w-20 h-20 mb-6 bg-gradient-to-br from-blue-500 to-purple-600 rounded-full">
+                <span className="text-3xl font-bold text-white">🏗️</span>
+              </div>
+              <h1 className="text-4xl sm:text-5xl font-bold bg-clip-text text-transparent bg-gradient-to-r from-blue-400 via-purple-400 to-pink-400 mb-2">
+                Builder
+              </h1>
+              <p className="text-slate-400 text-lg">Batch 22 Graduate</p>
+            </div>
+
+            {/* Bio */}
+            <div className="mb-8">
+              <p className="text-slate-300 text-center leading-relaxed">
+                Smart contract developer and blockchain enthusiast building on Arbitrum. Passionate about creating
+                secure, efficient Web3 infrastructure and exploring the potential of decentralized applications.
+              </p>
+            </div>
+
+            {/* Address Card */}
+            <div className="mb-8 p-4 bg-slate-700/50 border border-slate-600/50 rounded-lg">
+              <p className="text-slate-400 text-sm mb-2">Wallet Address</p>
+              <div className="flex items-center justify-between gap-4">
+                <code className="text-sm sm:text-base text-blue-400 font-mono break-all">{walletAddress}</code>
+                <button
+                  onClick={copyToClipboard}
+                  className="px-3 py-1 bg-blue-600 hover:bg-blue-700 text-white text-sm rounded-lg transition-colors flex-shrink-0"
+                >
+                  {copied ? "✓" : "Copy"}
+                </button>
+              </div>
+            </div>
+
+            {/* Skills */}
+            <div className="mb-8">
+              <h2 className="text-lg font-semibold text-white mb-4">Skills & Focus</h2>
+              <div className="grid grid-cols-2 sm:grid-cols-3 gap-3">
+                {["Solidity", "Web3", "Arbitrum", "Smart Contracts", "Security", "Testing"].map(skill => (
+                  <div
+                    key={skill}
+                    className="px-4 py-2 bg-slate-700/50 border border-slate-600/50 rounded-lg text-sm text-slate-300 text-center hover:bg-slate-600/50 transition-colors"
```

Comment:
You should only have hover effects for interactive elements. This hover effect misleads the user into thinking they can click it to navigate.